### PR TITLE
Vue best practices Refactor

### PR DESCRIPTION
**Pull request by AidanLaycock**

### CODE REVIEW

**Comment by AidanLaycock** [comment]
Hi @cherry-john & @nathanbillis - I got excited by the post, and I've done a quick overhaul of this component :) Feel free to tell me what you think and give me a shout if there is anything you'd like to changed further. 👍 

Please do try this before merging as I'm unable to test it at this time.

File: vue-activities-az.js [hunk 1]
```diff
@@ -4,213 +4,280 @@ let layout = `
...
+                    :alt="activity.name + ' Logo'"
+                  />
+              </div>
+               <div v-else>
+                      <div
+                        class="d-md-none justify-content-center"
+                        style="
+                          height: 5em;
+                          overflow: hidden;
+                          background-image: url('https://yusu.s3.eu-west-2.amazonaws.com/sums/website/images/placeholder-events.png');
```
Comment (nathanbillis) [inline vue-activities-az.js:127]: @cherry-john we need to update this placeholder image!

**Comment by nathanbillis** [comment]
Hi @AidanLaycock, Thank you for having a look 😄, we will give it a test and let you know. Just had a quick scan and it all looks good!

**Comment by AidanLaycock** [comment]
Amazing! There are definitely a few more refactors that could happen to this over time to make it as flexible as possible :) Happy to contribute further going forward, just let me know.

**Pull request closed without merging**